### ISSUE TITLE
Is the version of Python 3.5?

### ISSUE THREAD

**Issue by lucas1334533196** [tsunghan-wu/RandLA-Net-pytorch#31]
I have tried using Python 3.10, 3.8, and 3.7, but the compilation with the bash script 'compile_op.sh' failed for all of them. Only version 3.5 was successful. Does this project specifically require Python version 3.5 as well?

**Comment by mp0312** [comment]
thanks,i think the problem may driving by python version too,i will try it now

**Comment by kangzhehu** [comment]
yeah,I tried python 3.9,and the "compile_op.sh" also failed.So I changed the python verison to 3.6
